app/user/rest.py

```diff
@@ -475,7 +475,7 @@ def send_branding_request(user_id):
         notification_type=template.template_type,
         api_key_id=None,
         key_type=KEY_TYPE_NORMAL,
-        reply_to_text=service.get_default_reply_to_email_address(),
+        reply_to_text=None,
```

Comment:
Ditto. `reply_to_text` is not necessary.

Reply:
deleted